### PR TITLE
Update pt_br.json

### PR DESCRIPTION
**Pull request by Mikeliro**

### CODE REVIEW

File: src/main/resources/assets/simplemagnets/lang/pt_br.json [hunk 1]
```diff
@@ -12,13 +16,21 @@
...
-  "gui.advancedmagnet.whitelist.off": "Lista Negra",
+  "gui.advancedmagnet.whitelist.on": "Ignorar",
+  "gui.advancedmagnet.whitelist.off": "Pegar",
+  "gui.simplemagnets.demagnetization_coil.durability.on": "Combinar NBT",
+  "gui.simplemagnets.demagnetization_coil.durability.off": "Ignorar NBT",
+  "gui.simplemagnets.demagnetization_coil.range.increase": "Aumentar Alcance",
+  "gui.simplemagnets.demagnetization_coil.range.decrease": "Diminuir Alcance",
+  "gui.simplemagnets.demagnetization_coil.range": "Alcance ($number$x$number$x$number$ Blocos)",
   "keys.category.simplemagnets": "Simple Magnets",
   "keys.simplemagnets.toggle": "Alternar ímã"
```
Comment (SuperMartijn642) [inline src/main/resources/assets/simplemagnets/lang/pt_br.json]: I think there's a missing comma here. Kinda hard to see on my phone though.

Comment (Mikeliro) [reply src/main/resources/assets/simplemagnets/lang/pt_br.json]: Done

**Comment by SuperMartijn642** [comment]
Thank you! I'll merge it tomorrow.

**Comment by SuperMartijn642** [comment]
Thanks!

**Pull request merged**